recipe/meta.yaml

```
@@ -24,6 +24,7 @@ requirements:
     - hatch-vcs
   run:
     - python
+    - typing-extensions
```

https://github.com/hynek/structlog/blob/d38385cda711f62ad56018abfd72d4384f94e3ce/pyproject.toml#L30


```suggestion
    - typing-extensions # [py<311]
```